db/fixtures/ae_datastore/ManageIQ/System/Event/EmsEvent/RHEVM.class/user_add_vm_template_finished_failure.yaml

```diff
@@ -7,6 +7,4 @@ object:
     name: USER_ADD_VM_TEMPLATE_FINISHED_FAILURE
     inherits: 
     description: 
-  fields:
-  - rel4:
-      value: "/System/event_handlers/event_action_policy?target=src_vm&policy_event=vm_discover&param="
+  fields: []
```

Comment:
This file is configured for provider event USER_ADD_VM_TEMPLATE_FINISHED_FAILURE, still valid. I think we should leave it as a placeholder so that later we can hook with other policy events.